tests/cli/lib/lib.sh

```diff
@@ -16,14 +16,14 @@ if [ "$COMPOSER_TEST_FAIL_FAST" == "1" ]; then
   }
 fi
 
-export QEMU_BIN="/usr/bin/qemu-system-$(uname -m)"
+export QEMU_BIN="/usr/libexec/qemu-kvm"
```

Comment:
this change should go together with the 1st commit in the sequence

Reply:
OK, will move that to the 1st commit.